src/partials/features-zigzag.tsx

```diff
@@ -98,13 +98,27 @@ function FeaturesZigZag() {
                 data-aos="fade-up"
               >
                 <div className="max-w-full mx-auto md:max-w-none h-auto flex flex-row flex-wrap fill-[#1c1e21] dark:fill-[#e3e3e3]">
-                  <CloudbeesSvg className="flex-auto h-20 w-20 m-4" />
-                  <SplitSvg className="flex-auto h-20 w-20 m-4" />
-                  <FliptSvg className="flex-auto h-20 w-20 m-4" />
-                  <GoFeatureFlagSvg className="flex-auto h-20 w-20 m-4" />
-                  <FlagsmithSvg className="flex-auto h-20 w-20 m-4" />
-                  <LaunchDarklySvg className="flex-auto h-20 w-20 m-4" />
-                  <FlagdSvg className="flex-auto h-20 w-20 m-4" />
+                  <Link target="_blank" to="https://github.com/rollout/cloudbees-openfeature-provider-go">
```

Comment:
These links should be removed until [this issue](https://github.com/open-feature/openfeature.dev/issues/63) has been implemented.